pint/models/phase_jump.py

```diff
@@ -9,11 +9,14 @@
 
 
 class PhaseJump(TimingModel):
-    """This is a class to implement phase jumps 
+    """This is a class to implement phase jumps
     """
     def __init__(self):
         super(PhaseJump, self).__init__()
-        self.add_param(p.maskParameter(name = 'JUMP'))
+        # NOTE: it is dangours to put jump unit as second.
+        # Since JUMP is not always in the unit of time. 
```

Comment:
In TEMPO, JUMPs are always in time units, and are included as a time delay.  (They get added into the clock correction;  note, this is _not_ how we should do it here, just explaining how it is done in the tempo code.)

In TEMPO2, JUMPs are implemented the way @luojing1211 has done it here.

I think we should follow the TEMPO version, have JUMP specified in seconds and apply it as a delay (not phase).  I don't think there is anything dangerous about this, this seems totally straightforward to me.  As we've discussed before, we may also want to allow an actual phase jump (with units of pulse phase) using a different parameter name.